specification/agent/net/IHttpResourceLoadDetails.ts

```diff
@@ -38,6 +38,7 @@ export default interface IHttpResourceLoadDetails {
   responseTime?: number;
   responseTrailers?: IHttpHeaders;
   resourceType?: IResourceType;
+  resourceBodySize?: number;
```

Comment:
I think we're actually recording the "responseBodySize" in this PR. The "resourceBodySize" seems more akin to the un-gzipped asset, imo.

Reply:
Very much a discussion of semantics, but I do agree. It's anyway the responseBodySize I want, so fine for me to rename!